src/main/java/org/sagebionetworks/web/client/widget/evaluation/EvaluationRowWidget.java

```diff
@@ -131,6 +151,40 @@ public void configure(Evaluation evaluation, EvaluationActionHandler handler) {
 		submitToEvaluationButton.configure(null, submitToEvaluationParams, null, null);
 	}
 
+	@UiHandler("migrateButton")
+	public void migrateToEvaluationRound(ClickEvent event){
+		Modal modal = new Modal();
+		modal.setRemoveOnHide(true);
+		modal.setTitle("Migrate to Evaluation Round");
+		ModalBody body = new ModalBody();
+		Paragraph paragraph = new Paragraph();
+		paragraph.setText("Are you sure you want to perform the migration? This action is irreversible. ");
+		body.add(paragraph);
+		modal.add(body);
+		modal.show();
+		ModalFooter footer = new ModalFooter();
+		modal.add(footer);
+
+		Button cancelButton = new Button("Cancel", (cancelClickEvent)->{modal.hide();});
+		footer.add(cancelButton);
+
+		Button migrate = new Button("Migrate", (migrateClickEvent)->{synapseClient.migrateSubmissionQuotaToEvaluationRound(evaluation.getId(),
+				new AsyncCallback<Void>() {
+					@Override
+					public void onFailure(Throwable throwable) {
+						body.add(new Alert("ERROR migrating evaluation quota: \n\n" +throwable.getMessage(), AlertType.DANGER));
```

Comment:
This error is never cleared.  Recommend you inject a SynapseAlert into this class and always add it to the body.  Clear it before calling migrateSubmissionQuotaToEvaluationRound(), and on error tell it to "showError(throwable.getMessage())"